scripts/core/common.yml

```diff
@@ -552,7 +552,7 @@ members:
       desc: "[in] type of this structure"
     - type: "void*"
```

Comment:
Should this be `const void*`?

Reply:
@wdamon-intel : possibly, but with that, we are actually changing the type of this member with a new modifier, so we couldn't just change it w/o breaking compatibility. I would say let's just leave it as is, unless there's a strong justification.